markers_to_discard:
  - id: 15
  - id: 16
  - id: 17
  - id: 18

links:
  - id: 1
    parent_marker_id: 0
    child_marker_id: 1
  - id: 2
    parent_marker_id: 1
    child_marker_id: 2
  - id: 3
    parent_marker_id: 2
    child_marker_id: 3
  - id: 4
    parent_marker_id: 3
    child_marker_id: 4
  - id: 5
    parent_marker_id: 1
    child_marker_id: 5
  - id: 6
    parent_marker_id: 5
    child_marker_id: 6
  - id: 7
    parent_marker_id: 6
    child_marker_id: 7
  - id: 8
    parent_marker_id: 1
    child_marker_id: 8
  - id: 9
    parent_marker_id: 8
    child_marker_id: 9
  - id: 10
    parent_marker_id: 9
    child_marker_id: 10
  - id: 11
    parent_marker_id: 10
    child_marker_id: 11
  - id: 12
    parent_marker_id: 8
    child_marker_id: 12
  - id: 13
    parent_marker_id: 12
    child_marker_id: 13
  - id: 14
    parent_marker_id: 13
    child_marker_id: 14
  - id: 15
    parent_marker_id: 0
    child_marker_id: 15
  - id: 16
    parent_marker_id: 0
    child_marker_id: 16
  - id: 17
    parent_marker_id: 15
    child_marker_id: 17
  - id: 18
    parent_marker_id: 16
    child_marker_id: 18
  - id: 19
    parent_marker_id: 14
    child_marker_id: 19
  - id: 20
    parent_marker_id: 19
    child_marker_id: 20
  - id: 21
    parent_marker_id: 14
    child_marker_id: 21
  - id: 22
    parent_marker_id: 11
    child_marker_id: 22
  - id: 23
    parent_marker_id: 22
    child_marker_id: 23
  - id: 24
    parent_marker_id: 11
    child_marker_id: 24
